Added links from Flytelab (#652)
flyteorg/flytesnacks: cookbook/docs/weather_forecasting.rst

#####################
Weather forecasting
#####################

We will understand how **Flyte** can be used to build an end-to-end Online Machine Learning application that performs **Weather forecasting** using Pandera and Streamlit.

.. panels::
    :header: text-center
    :column: col-lg-12 p-2

    .. link-button:: https://github.com/flyteorg/flytelab/tree/main/projects/weather_forecasting
       :type: url
       :text: Weather forecasting Github repository
       :classes: btn-block stretched-link
    ^^^^^^^^^^^^
    Github repository of the weather forecasting project- an end-to-end online Machine Learning application

    ---

    .. link-button:: https://blog.flyte.org/building-a-weather-forecasting-application-with-flyte-pandera-and-streamlit
       :type: url
       :text: Detailed blog on weather forecasting
       :classes: btn-block stretched-link
    ^^^^^^^^^^^^
    Tutorial on building a weather forecasting model using Flyte, Pandera, and Streamlit

> .. toctree::
>     :maxdepth: -1
>     :caption: Contents
>     :hidden:
> 
>     https://github.com/flyteorg/flytelab/tree/main/projects/weather_forecasting    https://blog.flyte.org/building-a-weather-forecasting-application-with-flyte-pandera-and-streamlit

Reviewer: ```suggestion
.. toctree::
    :maxdepth: -1
    :caption: Contents
    :hidden:

    Github Repo <https://github.com/flyteorg/flytelab/tree/main/projects/weather_forecasting>
    Blog Post <https://blog.flyte.org/building-a-weather-forecasting-application-with-flyte-pandera-and-streamlit>
```

> ##########
> Weather forecasting
> ##########

Reviewer 2: ```suggestion
###############
Weather forecasting
###############
```

> An end-to-end Online Machine Learning application, built using Flyte, that performs **Weather forecasting** using Pandera and Streamlit.

Reviewer 2: ```suggestion
Learn how to build an online weather forecasting system that updates a model daily and displays hourly forecasts on a web UI, using Pandera and Streamlit.
```

.. panels::
    :header: text-center
    :column: col-lg-12 p-2

    .. link-button:: https://github.com/flyteorg/flytelab/tree/main/projects/weather_forecasting
       :type: url
       :text: Github repository
       :classes: btn-block stretched-link
    ^^^^^^^^^^^^

>     Github repository of the weather forecasting project: an end-to-end online Machine Learning application

Reviewer 2: ```suggestion
    The code can be found here.
```

    ---

    .. link-button:: https://blog.flyte.org/building-a-weather-forecasting-application-with-flyte-pandera-and-streamlit
       :type: url
       :text: Blog post
       :classes: btn-block stretched-link
    ^^^^^^^^^^^^

>     Tutorial on building a weather forecasting model using Flyte, Pandera, and Streamlit

Reviewer 2: ```suggestion
    An article detailing the architecture design of the weather forecasting application.
```
